packages/native/src/components/SearchBox/index.js

```diff
@@ -558,13 +566,20 @@ class SearchBox extends React.Component {
               ...defaultPopularSuggestionIcon(theme)
             })
           : null}
-        <Text
-          style={styles.itemText}
-          onPress={() => this.onSuggestionSelected(item)}
-          numberOfLines={1}
-        >
-          {item.label}
-        </Text>
+        {isPredictiveSuggestion ? (
+          <TouchableOpacity style={{display: 'flex', flexDirection: 'row', flex: 1}} onPress={() => this.onSuggestionSelected(item)}>
```

Comment:
@anik-ghosh-au7 Why are you using `TouchableOpacity`?

Reply:
You can wrap the content in a `View`.

Reply:
@bietkul I'm using `TouchableOpacity` here because I need the `onPress` prop to get the suggestion that is being selected by the user